playbooks/collect_activity/collect_activity.yml

```diff
@@ -15,13 +15,13 @@
       copy:
         src: stats.py
         dest: /tmp/collect_activity.py
-      become_user: www-data
+      become: yes
```

Comment:
@smarnach Yes, we should definitely try to keep these scripts running with the same user as the LMS/CMS uses at runtime.

`collect_activity.py` initializes Django in the LMS/CMS environment, and my main concern is that doing this as root may cause subtle issues down the line that are difficult to track down if, for example, the Django setup process caches anything to disk (which I suspect does happen, including with python bytecode caching, etc.), or if edX decides to add anything to the setup process that could result in files being written.